ui/app/orders/views/orderFulfillment.html

```diff
@@ -13,7 +13,8 @@ <h2 class="section-title">
                         <!--<span class="order-created-date">{{order.orderDate | bahmniDateTime}}</span>-->
                         <!--</span>-->
 
-                        <span ng-if="!order.hasObservations" class="new-order-highlight">New</span>
+                        <span ng-if="!order.hasObservations" class="order-label">New</span>
```

Comment:
"New" should come from Locale. Not hardcoded.

Reply:
Moved to locale